mucommander-core/src/main/java/com/mucommander/ui/main/DrivePopupButton.java

```diff
@@ -221,8 +221,12 @@ private void updateButton() {
 
             }
             setText(newLabel);
-            // Set the folder icon based on the current system icons policy
-            setIcon(FileIcons.getFileIcon(currentFolder));
+            { // local ctx for icon
+            // Set the folder icon slightly disobeying system icons policy for the current folder
+            Icon icon = FileIcons.hasProperSystemIcons() ?
+                    FileIcons.getSystemFileIcon(currentFolder) : FileIcons.getFileIcon(currentFolder);
```

Comment:
what's the reason for disobeying the system icons policy here?

Reply:
I was thinking that if somebody decided to have protocol icons then it work just like that (https://, gdrive:// etc).

Reply:
PR updated